code/modules/surgery/surgery.dm

```diff
@@ -10,7 +10,7 @@
 	var/list/possible_locs = list() 							//Multiple locations -- c0
 	var/ignore_clothes = 0										//This surgery ignores clothes
 	var/obj/item/organ/organ									//Operable body part
-
+	var/requires_missing_bodypart = 0							//Surgery available only when a bodypart is present, or only when it is missing.

```

Comment:
I just copypasted that var, I'll change it to use TRUE/FALSE. I'm taking suggestion for a better var name.